### PR TITLE
Gift Entry: Error Message feature when advanced mapping is disabled

### PR DESCRIPTION
**Pull request by voduyemi**
# Critical Changes

# Changes
Added an error message to disable use of the gift entry feature if Advanced Mapping has been disabled 
# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geTemplates/geTemplates.html [hunk 1]
```diff
@@ -9,7 +9,7 @@ <h1 slot="title">
         <div slot="actions"
              class="slds-button-group"
              role="group">
-            <lightning-button label="New Batch"
+            <lightning-button label="New Batch" if:true={isAccessible}
```
Comment (psadie-sfdo) [inline src/lwc/geTemplates/geTemplates.html]: @voduyemi - Could we move the `if:true` attribute here to its own line?

File: src/lwc/geTemplates/geTemplates.js [hunk 2]
```diff
@@ -41,10 +46,33 @@ export default class GeTemplates extends NavigationMixin(LightningElement) {
...
+        }
+    }
+
+    /*******************************************************************************
+     * @description Method checks for page level access. Currently only checks
+     * if Advanced Mapping is on from the Data Import Custom Settings.
+     */
+    checkPageAccess = async () => {
+        const dataImportSettings = await getDataImportSettings();
+        console.log('DataImportSetting '+ dataImportSettings[FIELD_MAPPING_METHOD_FIELD_INFO.fieldApiName]);
```
Comment (psadie-sfdo) [inline src/lwc/geTemplates/geTemplates.js]: Just gotta clear out this log.

File: src/lwc/geTemplates/geTemplates.js [hunk 3]
```diff
@@ -41,10 +46,33 @@ export default class GeTemplates extends NavigationMixin(LightningElement) {
...
+
+        const isAdvancedMappingOn =
+            dataImportSettings[FIELD_MAPPING_METHOD_FIELD_INFO.fieldApiName] === ADVANCED_MAPPING;
+        let hasPageAccess = false;
+        console.log('isAdvancedMappingOn ' +isAdvancedMappingOn);
```
Comment (psadie-sfdo) [inline src/lwc/geTemplates/geTemplates.js]: Just gotta clear out this log.

File: src/lwc/geTemplates/geTemplates.js [hunk 4]
```diff
@@ -41,10 +46,33 @@ export default class GeTemplates extends NavigationMixin(LightningElement) {
...
+        if (isAdvancedMappingOn) {
+            hasPageAccess = true;
+        } else {
+            this.isLoading = hasPageAccess =false;
```
Comment (psadie-sfdo) [inline src/lwc/geTemplates/geTemplates.js]:
```suggestion
            this.isLoading = hasPageAccess = false;
```

**Comment by voduyemi** [comment]
**lurch: attach W-037630

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HosiQAC" target="_blank">W-037630</a>

**Comment by melissabarber** [comment]
Hey @voduyemi @ptewson-sfdo & @kenlewis-sfdo I took a quick look at this in a scratch org, and the messaging looks good. Thanks!

**Pull request merged**